llvm/lib/Target/Hexagon/HexagonLoopIdiomRecognition.cpp

```diff
@@ -2246,6 +2247,11 @@ bool HexagonLoopIdiomRecognize::processCopyingStore(Loop *CurLoop,
       Type *PtrTy = PointerType::get(Ctx, 0);
       Type *VoidTy = Type::getVoidTy(Ctx);
       Module *M = Func->getParent();
+
+      // FIXME: This should check if the call is supported
+      StringRef HexagonVolatileMemcpyName =
```

Comment:
Can you get rid of the declaration here:

https://github.com/llvm/llvm-project/blob/d14d84075881ca24bf5890fa5eed31d1ea28b630/llvm/lib/Target/Hexagon/HexagonLoopIdiomRecognition.cpp#L108C1-L109C39